python/hail/expr/aggregators.py

```diff
@@ -647,50 +647,50 @@ def fraction(predicate) -> Float64Expression:
                           predicate._aggregations.push(Aggregation(predicate)))
 
 @typecheck(expr=agg_expr(expr_call))
-def hardy_weinberg(expr) -> StructExpression:
-    """Compute Hardy-Weinberg Equilbrium (HWE) p-value and heterozygosity ratio.
+def hwe_test(expr) -> StructExpression:
+    """Tests whether genotypes are in Hardy-Weinberg equilibrium.
 
     Examples
     --------
-    Compute HWE statistics per row of a dataset:
+    Test each row of a dataset:
 
-    >>> dataset_result = dataset.annotate_rows(hwe = agg.hardy_weinberg(dataset.GT))
+    >>> dataset_result = dataset.annotate_rows(hwe = agg.hwe_test(dataset.GT))
 
-    Compute HWE statistics for a single population:
+    Test each row on a sub-population:
 
     >>> dataset_result = dataset.annotate_rows(
-    ...     hwe_eas = agg.hardy_weinberg(agg.filter(dataset.pop == 'EAS', dataset.GT)))
+    ...     hwe_eas = agg.hwe_test(agg.filter(dataset.pop == 'EAS', dataset.GT)))
 
     Notes
     -----
-    This method returns a struct expression with the following fields:
+    This method performs the test described in :func:`.functions.hwe_test` based solely on
+    the three counts: ``n_hom_ref``, ``n_het``, and ``n_hom_var``.
```

Comment:
why are these in code? They're not parameters or output fields.

Reply:
switched to writing it out in english